### DISCUSSION TITLE
X11 VNX spécific port configuration

### DISCUSSION THREAD

**Discussion by RichardBachelier** [ltsp/ltsp#832] [General]
Hello,

I have created a service using systemd to launch x11vnc on a specific port (13179) instead of the default port 5900. This port is not assigned to any particular system program.

When I perform an nmap scan from my server to my client machine, the port 13179 remains closed. However, if I revert to the default port 5900, it becomes accessible.

I don't have any active firewalls, such as iptables or ufw.

Does Ltsp block any ports? Do I need to add a specific instruction?

Thank you for your response.

Best regards,

Richard

**Comment by alkisg** [comment]
Hi, no LTSP doesn't block any ports.

**Comment by RichardBachelier** [comment]
Thanks
